index.bs

```diff
@@ -190,7 +188,7 @@ The MIME Content Type for files with the ".avifs" file extension shall be "image
 
 This section defines the MIAF AV1 Baseline profile of [[!HEIF]], specifically for [[!AV1]] bitstreams, based on the constraints specified in [[!MIAF]] and identified by the brand  <dfn value export for="AV1 Image Item Type">MA1B</dfn>.
 
-If the brand <code>MA1B</code> is in the list of [=compatible_brands=] of a TrackTypeBox or [=FileTypeBox=], the common constraints in the Profiles and brands section shall apply.
+If the brand <code>MA1B</code> is in the list of [=compatible_brands=] of a [=TrackTypeBox=] or [=FileTypeBox=], the common constraints in the Profiles and brands section shall apply.
```

Comment:
This sentence mentions "Profiles and brands section". That could be changed into a hyperlink to that section.